### PR TITLE
add a7050 qx32 platform support

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: debian/syncd.init [hunk 1]
```diff
@@ -48,6 +48,10 @@ start)
         start_bcm
         DAEMON_ARGS+=" -p /etc/syncd.d/dell_s6100.profile -N"
 
+    elif [ "$aboot_platform" == "x86_64-arista_7050_qx32" ]; then
+        start_bcm
+        DAEMON_ARGS+=" -p /etc/syncd.d/arista_a7050_qx32.profile"
```
Comment (qiluo-msft) [inline debian/syncd.init]: Remove the leading blank? The same to above.

Comment (stcheng) [inline debian/syncd.init]: the blank line makes it clear to read?

Comment (qiluo-msft) [inline debian/syncd.init]: I mean the blank before "-p"

Comment (qiluo-msft) [inline debian/syncd.init]: Then the last branch is not consistent.

Comment (stcheng) [inline debian/syncd.init]: hmm yes. need to be consistent on adding space for this

**Pull request merged**